storage/aptosdb/src/db/include/aptosdb_writer.rs

```diff
@@ -247,30 +236,38 @@ impl AptosDB {
             //
             // TODO(grao): Consider propagating the error instead of panic, if necessary.
             s.spawn(|_| {
-                self.commit_events(chunk.first_version, chunk.transaction_outputs, skip_index_and_usage)
-                    .unwrap()
+                self.commit_events(
+                    chunk.first_version,
+                    chunk.transaction_outputs,
+                    skip_index_and_usage,
+                )
+                .unwrap()
             });
             s.spawn(|_| {
                 self.ledger_db
                     .write_set_db()
-                    .commit_write_sets(
+                    .commit_write_sets(chunk.first_version, chunk.transaction_outputs)
+                    .unwrap()
+            });
+            s.spawn(|_| {
+                self.ledger_db
+                    .transaction_db()
+                    .commit_transactions(
                         chunk.first_version,
-                        chunk.transaction_outputs,
+                        chunk.transactions,
+                        skip_index_and_usage,
                     )
                     .unwrap()
             });
             s.spawn(|_| {
                 self.ledger_db
-                    .transaction_db()
-                    .commit_transactions(chunk.first_version, chunk.transactions, skip_index_and_usage)
+                    .persisted_auxiliary_info_db()
+                    .commit_auxiliary_info(chunk.first_version, chunk.persisted_info)
                     .unwrap()
```

Comment:
Earlier, we were committing chunk.transactions, chunk.transaction_outputs.
Now, we are committing chunk.transactions and chunk.persisted_info.
We need to commit chunk.transaction_outputs too?

Reply:
ditto